cime_config/acme/machines/template.case.run

```diff
@@ -1,416 +1,327 @@
-        system(" $config{'CASETOOLS'}/getTiming -lid $LID ");
-        $logger->info( "gzipping timing stats..");
-        qx( gzip timing/acme_timing_stats.$LID );
-    }
-    chdir $owd;
-
-    # Save the timing files if desired
-    if($config{'SAVE_TIMING'} eq 'TRUE')
-    {
-        move( "$config{'RUNDIR'}/timing",  "$config{'RUNDIR'}/timing.$LID");
-    }
-}
-
-sub appendCaseStatus
-{
-    my $msg = shift;
-    open (my $CS, ">>", "$config{'CASEROOT'}/CaseStatus") or warn "cannot open $config{'CASEROOT'}/CaseStatus, $!";
-    print $CS "$msg\n";
-    close $CS;
-}
-
-# check to see if we need to do resubmission from this particular job,
-sub resubmitCheck()
-{
-#
-# Mira requires special logic
-#
-    if(($config{DOUT_S} eq 'FALSE' && $config{RESUBMIT} > 0) || ($config{MACH} eq "mira" && $config{DOUT_S} eq "TRUE")) {
-	my $submitscript = "$config{'CASEROOT'}/case.submit";
-	my $cwd = getcwd;
-	chdir $config{'CASEROOT'};
-	my $resubmitcommand = "$submitscript -resubmit -scriptname case.";
-
-	if(defined $config{TESTCASE} && $config{TESTCASE} eq "ERR"){
-	    $resubmitcommand .= "test";
-	}else{
-	    $resubmitcommand .= "run";
-	}
-	$logger->debug( "running resubmit check $resubmitcommand");
-	system($resubmitcommand);
-        if($?){ $logger->warn ("could not run ACME submit script, $! $?");}
-	chdir $cwd;
-    }
-}
-
-sub DoDataAssimilation
-{
-    my ($rundir, $caseroot, $da, $da_script, $lid, $logdir) = @_;
-
-    return unless($da eq "TRUE");
-
-    system(" $da_script 1> da.log.$lid 2>&1");
-    sleep(1);
-    $logger->logdie ("$da_script failed") unless ($? == 0);
-    chdir $rundir;
-    disposeLog($rundir,$caseroot, 'da' ,$lid,$logdir);
-    chdir $caseroot;
-}
+    # create the timing directories, optionally cleaning them if needed.
+    try:
+        os.stat(rundir)
+    except:
+        os.mkdir(rundir)       
+    if os.path.isdir(rundir + '/timing'):
+        shutil.rmtree(rundir + '/timing')
+    os.makedirs(rundir + '/timing/checkpoints')
+
+    # run preview namelists
+    preview_namelists()
+
+    # document process
+    appendCaseStatus(caseroot, "%s Run started \n" %(time.strftime("%Y-%m-%d %H:%M:%S")))
+
+    logger.info( "-------------------------------------------------------------------------")
+    logger.info( " - To prestage required restarts, untar a restart.tar file into %s" %(rundir))
+    logger.info( " - Case input data directory (DIN_LOC_ROOT) is %s " %(din_loc_root))
+    logger.info( " - Checking for required input datasets in DIN_LOC_ROOT")
+    logger.info( "-------------------------------------------------------------------------")
+
+###############################################################################
+def runModel(case):
+###############################################################################
 
+    # Set OMP_NUM_THREADS
+    num_threads = {{ thread_count }}
+    os.environ["OMP_NUM_THREADS"] = str(num_threads)
 
+    # Run the model
+    cmd = ("sleep 10")
```

Comment:
Agreed. Do you want to do this as part of this pull request? Or should we just create an issue that this should be done in a cleanup phase.